apps/src/code-studio/progressRedux.js

```diff
@@ -1,13 +1,16 @@
 /**
  * Reducer and actions for progress
  */
+import $ from 'jquery';
 import _ from 'lodash';
 import {makeEnum} from '../utils';
 import {mergeActivityResult, activityCssClass} from './activityUtils';
 import {LevelStatus, LevelKind} from '@cdo/apps/util/sharedConstants';
 import {TestResults} from '@cdo/apps/constants';
 import {ViewType, SET_VIEW_TYPE} from './viewAsRedux';
 import {processedLevel} from '@cdo/apps/templates/progress/progressHelpers';
+import {setVerified} from '@cdo/apps/code-studio/verifiedTeacherRedux';
+import {authorizeLockable} from './stageLockRedux';
 
 // Action types
 export const INIT_PROGRESS = 'progress/INIT_PROGRESS';
```

Comment:
This file is getting a bit large. Here is a short list of things that go together and could be easily extracted:
```
SET_USER_SIGNED_IN
SET_USER_TYPE
SET_IS_AGE_13_REQUIRED
```
one idea would be to extract these 3 actions and the state they apply to into a new userInfoRedux.js. 

Not required to do in this PR but please consider doing while you have some of this code fresh in your mind :-)